### PR TITLE
Adapt steam extract

### PR DESCRIPTION
**Pull request by Eibironke**
Adds search page to the dashboard

### CODE REVIEW

**Review by annalisev — COMMENTED** [review]
I think that the Dockerfile doesn't need updated

**Comment by Eibironke** [comment]
Changes need to be made, don't approve

**Comment by Eibironke** [comment]
> Changes need to be made, don't approve

All fixed now

File: pipeline/extract/steam/steam_extract.py [hunk 1]
```diff
@@ -150,7 +151,7 @@ def grab_all_games_details(all_web_containers: BeautifulSoup) -> list[list]:
         game_date = datetime.strptime(
             name_price_date_list[-1], "%d %b, %Y").date()
         name_price_date_list.pop(-1)
-        if game_date < date.today():
+        if game_date < (datetime.today() - timedelta(days=7)).date():
```
Comment (annalisev) [inline pipeline/extract/steam/steam_extract.py:154]: is this right?

Comment (Eibironke) [reply pipeline/extract/steam/steam_extract.py]: All sorted, thank you

File: pipeline/extract/steam/install-firefox.sh
Comment (annalisev) [inline pipeline/extract/steam/install-firefox.sh:1]: do we need this file?

**Review by annalisev — DISMISSED** [review]
Looks really good- this will be a great feature to have

**Pull request merged**